packages/nimble-components/src/table/components/group-row/styles.ts

```diff
@@ -1,34 +1,49 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
+import { White } from '@ni/nimble-tokens/dist/styledictionary/js/tokens';
 import {
+    applicationBackgroundColor,
     borderWidth,
     controlHeight,
     controlSlimHeight,
     fillHoverColor,
     mediumDelay,
     smallPadding,
-    standardPadding,
-    tableRowBorderColor
+    standardPadding
 } from '../../../theme-provider/design-tokens';
+import { Theme } from '../../../theme-provider/types';
+import { hexToRgbaCssColor } from '../../../utilities/style/colors';
+import { themeBehavior } from '../../../utilities/style/theme';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         align-items: center;
         height: calc(${controlHeight} + 2 * ${borderWidth});
-        border-top: calc(2 * ${borderWidth}) solid ${tableRowBorderColor};
-    }
-
-    :host(:hover) {
-        background: ${fillHoverColor};
+        border-top: calc(2 * ${borderWidth}) solid ${applicationBackgroundColor};
+        box-sizing: border-box;
+        background-clip: padding-box;
```

Comment:
Now that the border isn't semi-transparent, do we still need to set `background-clip` here?

Reply:
Nope!